### ISSUE TITLE
Is it possible to replace "RethinkDB instance" with Cloudant?

### ISSUE THREAD

**Issue by thomassuedbroecker** [IBM-MIL/IBM-Ready-App-for-Venue#13]
Hello,

I want to replace the "RethinkDB instance" in "Step 6: Create RethinkDB instancewith Cloudant".
Is the "RethinkDB" only used by using the adapters?
How much effort do you expect to transfer the app using Cloudant as sample DB?

Regards,
Thomas

**Comment by bweagle** [comment]
Yes, that's right. It should be very easy. You can see in the adapters the queries to the DB. You can simply replace those with Cloudant calls, once you have your DB.
